### ISSUE TITLE
'synda metric' not returning any result

### ISSUE THREAD

**Issue by jripsl** [Prodiguer/synda#43]
```
$ synda queue
status      count  size
done          132  343.4 GB
running        10  15.2 GB
waiting       174  464.0 GB 
```

```
$ synda metric
....
```

**Comment by jripsl** [comment]
Some informations are missing in the table.

Here is a fix for 3.3 and 3.4:

Run queries below

```
$ sqlite3 $HOME/sdt/db/sdt.db "update file set duration=cast(((julianday(end_date) - julianday(start_date)) * 86400.0) as integer) where status = 'done';"
$ sqlite3 $HOME/sdt/db/sdt.db "update file set duration=1 where duration=0 and status='done';"
$ sqlite3 $HOME/sdt/db/sdt.db "update file set rate=size/duration where status='done';"
```

'synda metric' command should work after that.

**Comment by jripsl** [comment]
Fixed in 3.5.
